django_ipam/tests/base/test_models.py

```diff
@@ -142,6 +142,26 @@ def test_invalid_master_subnet(self):
         if failed:
             self.fail('ValidationError not raised')
 
+    def test_correct_master_subnet_pass(self):
+        subnet1 = self._create_subnet(subnet='12.0.56.0/24')
+        try:
+            subnet2 = self._create_subnet(subnet='12.0.56.0/25', master_subnet=subnet1)
+            self._create_subnet(subnet='12.0.56.0/26', master_subnet=subnet2)
+        except ValidationError:
+            self.fail('Correct master_subnet not accepted')
+
+    def test_correct_master_subnet_fail(self):
+        failed = True
+        subnet1 = self._create_subnet(subnet='12.0.56.0/24')
+        try:
+            self._create_subnet(subnet='12.0.56.0/25', master_subnet=subnet1)
+            self._create_subnet(subnet='12.0.56.0/26', master_subnet=subnet1)
+        except ValidationError as e:
+            self.assertTrue(e.message_dict['subnet'] == ['Subnet overlaps with 12.0.56.0/25'])
+            failed = False
+        if failed:
```

Comment:
can you use the form `try/except/else` please?

Reply:
Should i make the change in this pull request or make change in all the test cases in one seperate pull request?

Reply:
I'd say make the change relative to this test in this PR, then in the other PR to increase coverage we can do more cleanup, what do you think?